tool/src/js/formatter.rs

```diff
@@ -100,10 +100,19 @@ impl<'tcx> JSFormatter<'tcx> {
     }
 
     /// Just creates `/** */` doc strings.
-    pub fn fmt_docs(&self, docs: &Docs) -> String {
+    pub fn fmt_docs(&self, docs: &Docs, deprecated: Option<&str>) -> String {
```

Comment:
suggestion: can the deprecated field be made a part of `docs`?

Reply:
I'm not sure if it should, but overall it seems docs-adjacent.

Reply:
I think it makes more sense in `Attrs`.

Reply:
I could pass `Attrs` here, that would also cover the experimental attr in the future.